examples/3d/3d_scene.rs

```diff
@@ -1,3 +1,5 @@
+//! Simple 3D scene with basic shapes and lighting
```

Comment:
Overall, I would think about changing the description into something less generic, that makes the reader expect more what they are about to see, since the focus of the example is on the scene itself; maybe something like:

> A simple 3D scene with light shining over a cube sitting on a plane.